config/application.properties

```diff
@@ -1,3 +1,11 @@
+# oauth information
+google.oauth.client.id = ${GOOGLE_OAUTH_CLIENT_ID}
+google.oauth.client.secret = ${GOOGLE_OAUTH_CLIENT_SECRET}
+
+# hmhb token information (used for generating jwt tokens)
+hmhb.jwt.domain = hmhb.herokuapp.com
```

Comment:
This is used for the audience in the jwt token.  The audience is something I validate.  Right now I've just configured this to be hmhb.herokuapp.com (domain probably isn't a good name for this).

At one point, I considered looking at the domain for this, however, that probably doesn't make sense.

If I'm going to pull the domain, should I pull it out of the request?  What if we have multiple domains in the future (or more specific domain names app1.hmhb.herokuapp.com)?  It wouldn't make sense that authentication fails due to something like that.

So, for now, it's a static thing.  This has a fun perk of my jwt token working on any of our instances (hmhb, hmhb-ryan, hmhb-mckoon, localhost).  That's probably how it should be.